examples/4_mine_and_craft.py

```diff
@@ -201,11 +201,11 @@ def mineWhileInRange(rob):
     rob.sendCommand('attack 0')
 
 # A higher-level skill for getting sticks
-def getSticks(rob):
+def getSticks(rob, minelogy_instance):
```

Comment:
It's a subjective preference, but `minelogy_instance` looks a little bit awkward. I'd prefer something more compact... I don't know - `mlogy`, `mclogy`, `mckb`.

Reply:
> It's a subjective preference, but `minelogy_instance` looks a little bit awkward. I'd prefer something more compact... I don't know - `mlogy`, `mclogy`, `mckb`.

Done